cypress/support/step_definitions/api_admin_update_registry_metadata_.js

```diff
@@ -51,7 +51,7 @@ then('the metadata for the entity registry is updated', () => {
 					'api-key': apiKey
 				}
 			}).then((response) => {
-				expect(response.body.metadata.description).to.equals('updatedDescriptionValue')
+				expect(response.body).to.have.string('updatedDescriptionValue')

```

Comment:
We need the wider discussion of Cypress and test methodology, which I suspect is a Jira task for the first week in January